src/strategies/prepo-vesting/index.ts

```diff
@@ -8,7 +8,8 @@ export const version = '1.0.0';
 const abi = [
   'function getAmountAllocated(address _recipient) external view override returns (uint256)',
   'function getClaimableAmount(address _recipient) public view override returns (uint256)',
-  'function getVestedAmount(address _recipient) public view override returns (uint256)'
+  'function getVestedAmount(address _recipient) public view override returns (uint256)',
+  'function isPaused() external view returns (bool)'
```

Comment:
```suggestion
  'function isPaused() external view override returns (bool)'
```
adding `override` because the other ones above also include it.
do other strategies also include the `override` keyword in the abi?

Reply:
I just realize no other projects included override, I'll remove it in ours